### PR TITLE
client: client test need to be check

### LINKED ISSUE #6781: client: client test need to be check
### What did you do?
When I check code for test https://github.com/tikv/pd/blob/master/client/resource_group/controller/model_test.go#L41-L46
must fail, but was not tested.

### What did you expect to see?
test ci all

### What did you see instead?
must fail, but was not tested.

### What version of PD are you using (`pd-server -V`)?
master

### PR DESCRIPTION
**Pull request by HuSharp**
### What problem does this PR solve?

Issue Number: Close #6781

### What is changed and how does it work?

When I check code for test [master/client/resource_group/controller/model_test.go#L41-L46](https://github.com/tikv/pd/blob/master/client/resource_group/controller/model_test.go?rgh-link-date=2023-07-10T07%3A31%3A07Z#L41-L46)
must fail, but was not tested.

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

### Release note

```release-note
None.
```

### CODE REVIEW

File: client/resource_group/controller/model_test.go [hunk 1]
```diff
@@ -69,18 +61,6 @@ func TestGetRUTokenBucketSetting(t *testing.T) {
 	if result != expected {
 		t.Errorf("Expected nil but got %v", result)
 	}
-
```
Comment (lhy1024) [inline client/resource_group/controller/model_test.go:72]: Why remove them?

Comment (HuSharp) [reply client/resource_group/controller/model_test.go:72]: stale check, now only `RequestUnitType_RU` type is available cc @nolouch

**Review by JmPotato — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Pull request merged**